src/styles/components/_dropdown.scss

```diff
@@ -32,6 +32,7 @@ div.p-dropdown {
 }
 
 .p-multiselect {
+  z-index: 1000 !important; 
```

Comment:
straightforward fix as z-index for the multiselect just needs to be higher than the tooltip